cypher/cypher-docs/src/docs/dev/syntax/operators.asciidoc

```diff
@@ -1,10 +1,10 @@
-[[query-operators-property]]
-== Property operators
+[[query-operators-property-deprecated]]
+== Deprecated property operators
 
 [NOTE]
```

Comment:
If this was removed in 2.0, how much longer should we carry this note on in the documentation?

Reply:
Does Cypher give a warning on this usage? If so I think we should just remove this. If not, well maybe it could? Either case I think this should go soon.